components/article/content.tsx

```
@@ -292,6 +296,8 @@ export const ArticleContent: React.FC<ArticleContentProps> = ({
                 ) : data.article?.content ? (
                   <div
                     className="mt-6 wrap-break-word prose dark:prose-invert max-w-none"
+                    dir={data.article.dir || 'ltr'}
+                    lang={data.article.lang || undefined}
                     dangerouslySetInnerHTML={{ __html: data.article.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)